projects/pydateutil/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://dateutil.readthedocs.io/en/stable/"
+language: python
+primary_contact: "paul@ganssle.io"
```

Comment:
Did the maintainer approve? iirc there was no maintainer response -- if this is correct please remove and then use `vendor_ccs` as list of `ccs`, and could you please include me in the `vendor_ccs`?

Reply:
Like this ?

Reply:
Yes! Thank you